src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,13 +3,36 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int STORAGE_CAPACITY = 10;
+    private static final int ELEMENT_SIZE = 2;
+    private static final int KEY = 0;
+    private static final int VALUE = 1;
+    private final Object[][] array = new Object[STORAGE_CAPACITY][ELEMENT_SIZE];
 
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < STORAGE_CAPACITY; i++) {
+            if (array[i][VALUE] == null) {
```

Comment:
```suggestion
            if (array[i][VALUE] == null) {
```
What if I had the null value by some real key? This key and value will be overridden